openmdao/docs/advanced_guide/derivs/partial_derivs_explicit.rst

```diff
@@ -0,0 +1,83 @@
+.. _advanced_guide_partial_derivs_explicit:
+
+OpenMDAO considers component derivatives to be **partial-derivatives**.
+The framework uses these partial-derivatives in order to compute the **total-derivatives** across your whole model.
+This tutorial is focused on how to define the partial-derivatives* for instances of :ref:`ExplicitComponent <comp-type-2-explicitcomp>`.
+
+
+
+****************************************************
+Defining Partial-Derivatives on Explicit Components
+****************************************************
+
+For any :ref:`ExplicitComponent <comp-type-2-explicitcomp>` instance you are going to provide derivatives of the **outputs with respect to the inputs**.
+Whenever you are going to define derivatives, there are two things you're required to do:
+
+    #. delclare the partial-derivatives via :code:`declare_partials`
+    #. specify their values via :code:`compute_partials`
+
+Here is a example, based on the :ref:`Betz Limit Example <betz_limit_tutorial>`:
+
+.. embed-code::
+    openmdao.test_suite.test_examples.test_betz_limit.ActuatorDisc
+
+
+The calls to :code:`declare_partials` tell OpenMDAO which partial derivatives to expect.
+This is always done inside the setup method.
+In this example not all the outputs depend on all the inputs, and you see that if you look at the derivative declarations.
+Any partial that is not declared is assumed to be 0.
+You could have, instead declared all the partials in just one line as follows (see this :ref:`feature doc <feature_specify_partials>` for more details):
+
+.. code::
+
+    self.declare_partials('*', '*')
+
+However, that would mean that OpenMDAO thinks that all the partials are non-zero.
+While you saved yourself a few lines of code there, it would come at the potential expense of performance.
+So generally its better to be more specific and declare only the non-zero partials.
+
+.. note::
+    There are a few more options to :code:`declare_partials` that are worth taking a look at.
+    There is support for when your derivatives are constant and if you want to specify derivatives in a sparse AIJ format.
+    The full details are :ref:`here <feature_specify_partials>`.
+
+After you declare the non-zero partial derivatives, you need to implement the :code:`compute_partials` method to perform the actual
+derivative computations.
+OpenMDAO will call this method whenever it needs to work with the partial derivatives.
+The values are stored in the Jacobian object, :code:`J`, and get used in the linear solutions that are necessary to compute model level total-derivatives.
+This API results in the actual assembly of the jacobian matrix in memory.
```

Comment:
Suggest removing the word "actual".  
Also suggest moving these two sentences to the next paragraph.